api/src/main/resources/workbench-api.yaml

```diff
@@ -4251,6 +4251,21 @@ paths:
       responses:
         200:
           description: 'The survey responses were successfully stored.'
+
+  "/v1/admin/workspaces/{workspaceNamespace}/disks":
+    parameters:
+    - "$ref": "#/parameters/workspaceNamespace"
+    get:
+      tags:
+      - disks
```

Comment:
usually we use different controller for Admin endpoints, e.g. worksapceAdmin, runtimeAdmin. Shall we still follow that pattern?

Reply:
Good call. I updated this.